src/core/pointcloud/qgspointclouddataprovider.h

```diff
@@ -211,6 +239,14 @@ class CORE_EXPORT QgsPointCloudDataProvider: public QgsDataProvider
      * providers will return NULLPTR.
      */
     virtual QgsPointCloudRenderer *createRenderer( const QVariantMap &configuration = QVariantMap() ) const SIP_FACTORY;
+
+    /**
+     * Returns whether the dataset contains statistics metadata
+     *
+     * \since QGIS 3.26
+     */
+    virtual bool containsStatisticsMetadata() const;
```

Comment:
I would suggest `hasStatisticsMetadata()` - it's shorter and more common to use "has" in Qt apis